content/en/dashboards/querying/_index.md

```diff
@@ -174,6 +174,16 @@ status:error / status:info
 
 **Note**: Formulas are not lettered. Arithmetic cannot be done between formulas.
 
+Here is an example using the `max` operator to find the maximum value between `status:error` and `status:ok`.
+
+```text
+max(status:error, status:ok)
```

Comment:
Racheal, let's have a query that is either both metrics or metrics + another data source here

Reply:
Thanks for pointing that out @ijkaylin, this is the example I am planning to use and seen in our customer FR board. I am querying for the avg CPU usage within 10-second intervals for two zones and then identifying the maximum value between the two zones for each rollup which can be used for resource optimization. 

<img width="956" alt="Screenshot 2023-10-13 at 1 40 22 PM" src="https://github.com/DataDog/documentation/assets/144172411/09be48be-0814-4279-8bdd-b375b72001a1">